lib/FrameManager.js

```diff
@@ -405,7 +406,28 @@ class Frame {
       document.head.appendChild(style);
     }
   }
+  /**
+  *@param {string} selector
+  *@param {!Array<string>} values
+   */
+
+  async select(selector, ...values){
```

Comment:
`page.select` should now use main frame's select:

```js
class Page {
  // ...
  async select(selector, ...values) {
    return this.mainFrame().select(selector, ...values);
  }
  // ...
}
```
Use `page.mainFrame().select()`